src/main/java/com/amihaiemil/docker/RtLogs.java

```diff
@@ -153,12 +149,18 @@ public Reader follow()
 
     @Override
     public Logs stdout() throws IOException, UnexpectedResponseException {
-        return new RtLogs(this.owner, this.client, this.baseUri, true, false);
+        final Map<String, String> params = new HashMap<>();
```

Comment:
@bkuzmic I think here and bellow, before putting the ``stdout`` and ``stderr`` we should ``.putAll(this.options)``, right? So we don't override previously set options.

Reply:
@amihaiemil I agree. Right now, you can't really change the options (which is an empty Map), but it makes sense when the constructor will become public.

Reply:
@bkuzmic I wouldn't make the ctor public. Instead, I would add methods (fluent methods, like ``stderr()`` and ``stdout()`` to specify options -- it's more intuitive I guess.

Reply:
@amihaiemil I like fluent methods also. I'll leave the ctor as is and since there is already an existing puzzle to continue, it can be done in another job.